config/application.rb

```diff
@@ -56,8 +56,6 @@ class Application < Rails::Application
     # Rewrite FoxitSDK relative routes to redirect to /assets
     config.middleware.insert_before(Rack::Runtime, Rack::Rewrite) do
       r301 %r{lib/uix-addons/(.*)}, "/assets/lib/uix-addons/$1"
-      r301 %r{admin/source_files/([^/]*)/redact_file/lib/(.*)}, "/assets/lib/$2"
```

Comment:
@littleforest heads up: I'll test in staging the PDF redaction feature since we don't have tests for the Foxit SDK plugin

Reply:
Great, thanks!